libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -343,9 +333,6 @@ class MapboxNavigation(
         tripSession.unregisterAllVoiceInstructionsObservers()
         tripSession.route = null
 
-        // TODO replace this with a destroy when nav-native has a destructor
-        navigator.create(navigationOptions.deviceProfile, navigatorConfig, logger)
```

Comment:
What was the test case for this again? Why does onDestroy need to reset the navigator?